packages/seal-opensearch-adapter/composer.json

```diff
@@ -41,7 +41,8 @@
         "phpstan/phpstan": "^1.10",
         "phpstan/phpstan-phpunit": "^1.3",
         "phpunit/phpunit": "^10.3",
-        "rector/rector": "^1.0"
+        "rector/rector": "^1.0",
+        "symfony/http-client": "^5.4 || ^6.0 || ^7.0"
```

Comment:
else it did automatically add 7.3-x when run `composer lint` adding it directly seems tow ork